validator/remote/src/main/java/tech/pegasys/teku/validator/remote/typedef/handlers/CreateBlockRequest.java

```diff
@@ -85,12 +84,18 @@ public Optional<BeaconBlock> createUnsignedBlock(
       final BLSSignature randaoReveal, final Optional<Bytes32> graffiti) {
     final Map<String, String> queryParams = new HashMap<>();
     queryParams.put("randao_reveal", randaoReveal.toString());
+    final Map<String, String> headers = new HashMap<>();
     graffiti.ifPresent(bytes32 -> queryParams.put("graffiti", bytes32.toHexString()));
 
+    if (preferSszBlockEncoding) {
+      headers.put("Accept", "application/octet-stream");
```

Comment:
We should make this explicitly state that we'd prefer octet-stream but will accept json.  Otherwise we run the risk of some clients not automatically falling back to JSON.